src/_posts/platform/deployment/2000-01-01-deploy-with-github.md

```diff
@@ -13,16 +13,14 @@ Enterprise. You will have to link it to a GitHub repository and branch.
 
 ### Using the Web Dashboard
 
-In the "Code" section in your dashboard, you'll find a category *Deploy with
-GitHub*. Please click on `Link this app to GitHub` and check you are in the
-right account. Choose your GitHub user, and find your GitHub repository in the
-list or with the search bar.
+In the "Deploy → Configuration" section in your dashboard, you find a category *GitHub*. Please click on it then `Connect account` and check you are in the right account. Choose your *GitHub user* or your *organization* and find your GitHub repository in the list or with the search bar.
```

Comment:
The part about connecting to GitHub account should be done in the account settings IMO: https://dashboard.scalingo.com/account/integrations. And we could give directly the link to the dashboard.